src/stimulus/data/encoding/encoders.py

```diff
@@ -13,15 +16,6 @@
 
 class AbstractEncoder(ABC):
     """Abstract class for encoders.
```

Comment:
Shouldn't Abstract classes also be documented ? see [here](https://gemseo.readthedocs.io/en/3.2.1/software/example_google_docstring.html?t )

Reply:
what would you write there?

Reply:
In theory, the methods to be implemented by the child classes and their expected input and outputs + types. 
This is done so that if a childclass is implemented without docstring, this docstring is used by default